src/assets/githubAction/githubActionBeginner.md

```diff
@@ -0,0 +1,130 @@
+---
+title: Introduction to GitHub Actions - CI & CD
+authorName: Nabin Ale
+authorAvatar : https://avatars.githubusercontent.com/u/61624650?s=400&u=8f992a9ca189af19c40cb070a5af1e02669d3884&v=4
+authorLink: https://github.com/nabim777
+createdAt: Apr 22, 2024
+tags: CI, CD, github, githubAction
```

Comment:
```suggestion
tags: continuous integration, continuous delivery, continuous deployment, github actions, ci, cd
```
The tag will be used to search and filter the content. People will generally search the content with space-separated words so I prefer to use either a single keyword or space-separated words.